$MirOS: ports/print/cups/patches/patch-filter_Makefile,v 1.4 2006/11/08 18:25:45 bsiegert Exp $
--- filter/Makefile.orig	Sun Dec 17 02:43:58 2006
+++ filter/Makefile	Thu Mar 15 19:18:39 2007
@@ -180,7 +180,7 @@ apihelp:
 
 formtops:	$(FORMOBJS) common.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ $(FORMOBJS) common.o $(LIBS) -lm
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ $(FORMOBJS) common.o $(LIBS) -lm
 
 
 #
@@ -189,7 +189,7 @@ formtops:	$(FORMOBJS) common.o ../cups/$
 
 gziptoany:	gziptoany.o ../Makedefs ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ gziptoany.o $(LIBZ) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ gziptoany.o $(LIBZ) $(LIBS)
 
 
 #
@@ -198,7 +198,7 @@ gziptoany:	gziptoany.o ../Makedefs ../cu
 
 hpgltops:	$(HPGLOBJS) common.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ $(HPGLOBJS) common.o $(LIBS) -lm
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ $(HPGLOBJS) common.o $(LIBS) -lm
 
 
 #
@@ -290,8 +290,8 @@ libcupsimage.a:	$(IMAGEOBJS)
 
 testimage:	testimage.o libcupsimage.a ../Makedefs
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testimage.o libcupsimage.a \
-		$(IMGLIBS) $(DSOLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testimage.o $(LINKCUPSIMAGE) \
+		$(IMGLIBS) $(LIBS)
 
 
 #
@@ -301,7 +301,7 @@ testimage:	testimage.o libcupsimage.a ..
 imagetops:	imagetops.o common.o $(LIBCUPSIMAGE) \
 		../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ imagetops.o common.o $(LINKCUPSIMAGE) \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ imagetops.o common.o $(LINKCUPSIMAGE) \
 		$(IMGLIBS) $(LIBS)
 
 
@@ -312,7 +312,7 @@ imagetops:	imagetops.o common.o $(LIBCUP
 imagetoraster:	imagetoraster.o common.o $(LIBCUPSIMAGE) \
 		../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ imagetoraster.o common.o $(LINKCUPSIMAGE) \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ imagetoraster.o common.o $(LINKCUPSIMAGE) \
 		$(IMGLIBS) $(LIBS)
 
 
@@ -322,7 +322,7 @@ imagetoraster:	imagetoraster.o common.o 
 
 pstops:	pstops.o common.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ pstops.o common.o $(LIBS) -lm
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ pstops.o common.o $(LIBS) -lm
 
 
 #
@@ -331,7 +331,7 @@ pstops:	pstops.o common.o ../cups/$(LIBC
 
 rastertolabel:	rastertolabel.o ../cups/$(LIBCUPS) $(LIBCUPSIMAGE)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ rastertolabel.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ rastertolabel.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
 
 
 #
@@ -340,7 +340,7 @@ rastertolabel:	rastertolabel.o ../cups/$
 
 rastertoepson:	rastertoepson.o ../cups/$(LIBCUPS) $(LIBCUPSIMAGE)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ rastertoepson.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ rastertoepson.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
 
 
 #
@@ -349,17 +349,17 @@ rastertoepson:	rastertoepson.o ../cups/$
 
 rastertohp:	rastertohp.o ../cups/$(LIBCUPS) $(LIBCUPSIMAGE)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ rastertohp.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ rastertohp.o $(LINKCUPSIMAGE) $(IMGLIBS) $(LIBS)
 
 
 #
 # testraster
 #
 
-testraster:	testraster.o ../cups/libcups.a libcupsimage.a
+testraster:	testraster.o ../cups/$(LIBCUPS) $(LIBCUPSIMAGE)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testraster.o libcupsimage.a ../cups/libcups.a \
-		$(IMGLIBS) $(DSOLIBS) $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testraster.o $(LINKCUPSIMAGE) \
+		$(IMGLIBS) $(LIBS)
 
 
 #
@@ -368,7 +368,7 @@ testraster:	testraster.o ../cups/libcups
 
 rasterbench:	rasterbench.o raster.o
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ rasterbench.o raster.o
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ rasterbench.o raster.o
 
 
 #
@@ -378,7 +378,7 @@ rasterbench:	rasterbench.o raster.o
 texttops:	texttops.o textcommon.o common.o \
 		../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ texttops.o textcommon.o common.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ texttops.o textcommon.o common.o $(LIBS)
 
 
 #
